structure.go

```diff
@@ -125,7 +125,6 @@ type MediaPlaylist struct {
 	count            uint // number of segments added to the playlist
 	buf              bytes.Buffer
 	ver              uint8
-	Key              *Key // EXT-X-KEY is optional encryption key displayed before any segments (default key for the playlist)
```

Comment:
Why cannot you let this attribute stay and have a value of `nil` so that it is not rendered?